lib/inherit.js

```diff
@@ -84,7 +79,14 @@ function override(base, res, add) {
 
                         return res;
                     };
-                result.__base = baseMethod;
+
+                result.__base = function() {
+                    var baseMethod = base[name]
```

Comment:
> I agree it's not an ideal solution but do you really do async supercalls often?

From time to time. As often as I do supercalls at all

> It makes you repeat a method name from a base class

Yes, and it increases readability. Also it makes possible to call other method from base class (but it is weird case anyway)

> In languages that support supercalls natively you don't have to repeat it

How about `C++`?
```cpp
void Inherited::method(int arg) {
    Base::method(arg);
}
```